docs/package-inclusion-guide.md

```diff
@@ -0,0 +1,55 @@
+# Azure Cloud Shell Package Inclusion Guide
+
+## Background
+
+There are millions (or potentially billions) of Linux packages. This guide goes through the principles of what the Azure Cloud Shell team uses to consider what gets included in the service.
+
+
+
+## General Principles
+
+1. Useful for Azure users and administrators 
+2. Available on [Azure Linux Operating System](https://github.com/microsoft/azurelinux) 
+3. Vulnerability free 
+    - No unfixed CVEs for the package on Azure Linux OS past 30 days 
```

Comment:
```suggestion
3. Is consistently vulnerability free:
    - Does not have unfixed CVE lingering around. 
```

I think we already made a point that it has to be a package on Azure Linux, so we don't have to be super specific here.